indico/MaKaC/conference.py

```diff
@@ -830,10 +830,10 @@ def unindexConf(self, conf):
         conf.unindexConf()
 
     @unify_user_args
-    def newConference(self, creator, title, start_dt, end_dt, timezone):
+    def newConference(self, creator, title, start_dt, end_dt, timezone, location_data):
```

Comment:
why here? you can set it later - the reason why some attributes are set here is because they are required and creation of the Event object would fail otherwise